Documentation/project-docs/performance-tests.md

```diff
@@ -1,47 +1,29 @@
 
-Requirements
---------------------
-### Windows
-To run performance tests on Windows, .NET portable v5.0 is required. This library is included in [the Visual Studio Community 2015 download](https://www.visualstudio.com/products/visual-studio-community-vs). To get the correct packages during installation, follow these steps after opening the installer:
-1. Select "Custom Installation" if no installation is present, or "Modify" otherwise
-2. Check the "Universal Windows App Development Tools" box under the "Windows and Web Development" menu
-3. Install
-### Linux
-Performance tests on Linux require all of the same steps as they do for regular xunit tests - see the linux instructions [here](https://github.com/dotnet/corefx/blob/master/Documentation/building/unix-instructions.md). Once you can have a directory on your Linux machine with a working corerun and xunit.console.netcore.exe (as well as the test dll containing your perf tests!), you only need to run the following command:
-
-`dnu commands install Microsoft.DotNet.xunit.performance.runner.dnx 1.0.0-alpha-build0021 -f https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json`
-
-Be careful that your mscorlib, libcoreclr, and test dlls were compiled using the "/p:Configuration=Release" property. Otherwise you may get skewed results.
-
-Running the tests
+Building and Running Tests
 -----------
-### Windows
-Performance test files (if present) are stored within a library's ```tests/Performance``` directory and contain test methods that are all marked with a perf-specific *Benchmark* attribute. The performance tests will only be run if the ```performance``` property is set to ```true```.
+Performance test files (if present) are stored within a library's ```tests/Performance``` directory and contain test methods that are all marked with a perf-specific *Benchmark* attribute.
 
-Before running the performance tests you must run ```build -release``` from the root folder.
+**Step # 1:** Prior to running performance tests, a full build from the repo root must be completed: ```build -release```
 
-To build and run the tests using msbuild for a project, run ```msbuild /t:BuildAndTest /p:Performance=true /p:ConfigurationGroup=Release /p:TargetOS=Windows_NT``` from the Performance directory with Admin privileges. If the v5.0 assemblies aren't installed on your system, an error will be raised and no tests will be run.
+**Step # 2:** Change directory to the performance tests directory: ```cd path/to/library/tests/Performance```
 
-Note: Because build.cmd runs tests concurrently, it's not recommended that you execute the perf tests using it.
+**Step # 3:** Build and run the tests:
+ - Windows (using admin command shell): ```msbuild /t:BuildAndTest /p:Performance=true /p:ConfigurationGroup=Release /p:TargetOS=Windows_NT```
+ - Linux: ```msbuild /t:BuildAndTest /p:Performance=true /p:ConfigurationGroup=Release /p:TargetOS=Linux```
```

Comment:
From https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md

> **Note:** If building in a non-Windows environment, call `<repo-root>/Tools/msbuild.sh` instead of just `msbuild`.

Should we update the command here to match that?

Reply:
Yes, we should - I knew I forgot something...  Thanks for pointing this out.